.github/workflows/cli-release.yml

```diff
@@ -23,23 +23,32 @@ permissions:
   pull-requests: read
   id-token: write # Needed for aws-actions/configure-aws-credentials@v1
 
-
 jobs:
   tests:
     uses: ./.github/workflows/cli-tests.yaml
     with:
       run-mac-tests: true
 
-  prerelease:
+  weekly:
     runs-on: ubuntu-latest
     environment: release
     needs: tests
-    if: ${{ inputs.is_prerelease || github.event.schedule }}
+    if: ${{ inputs.update_weekly || github.event.schedule }}
     steps:
       - name: Checkout source code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0 # Needed by goreleaser to browse history.
+      - name: Determine weekly tag
+        # This tag is semver and works with semver.Compare
+        run: echo "WEEKLY_TAG=0.0.0-weekly.$(date +%Y.%U)" >> $GITHUB_ENV
```

Comment:
Could we do `0.0.0-dev.2023-04-03`, which should be:
`$(date +%Y-%m-%d)`

Open to changing `dev` to `rc`, `alpha`, `edge` or something similar ... but trying to avoid promising a particular cadence in the tag.

Rationale:
1. Even though we are choosing to run this weekly, I prefer to not promise weekly in the name. That way if we later want to change to nightly, or something else, we don't need to change how the tags get generated.

2. For the same reason, switching to Month + Day over Week number gives us the flexibility to do multiple releases in a week if we ever need to.

3. `YYYY-MM-DD` over `YYYY.MM.DD` because the first one matches the ISO 8601 standard, and I think we should err on choosing the standard.

Reply:
nit:
- `edge` would be more suitable for this IMO
- `trunk` is usually specifically referred to as the version of `main` branch
- `alpha` has the downside that it builds the expectation of a `beta` version, etc.